src/crawler.ts

```diff
@@ -1555,69 +1568,45 @@ self.__bx_behaviors.selectMainBehavior();
     const waczFilename = this.params.collection.concat(".wacz");
     const waczPath = path.join(this.collDir, waczFilename);
 
-    const createArgs = [
-      "create",
-      "-o",
-      waczPath,
-      "--pages",
-      this.seedPagesFile,
-      "--extra-pages",
-      this.otherPagesFile,
-      "--copy-pages",
-      "--log-directory",
-      this.logDir,
-    ];
+    const waczLogger = new WACZLogger(logger);
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const waczOpts: Record<string, any> = {
+      input: warcFileList.map((x) => path.join(this.archivesDir, x)),
+      output: waczPath,
+      pages: this.pagesDir,
+      detectPages: false,
+      indexFromWARCs: false,
+      logDirectory: this.logDir,
```

Comment:
Seems like `logDirectory` may not be working as expected, not seeing log files in the resulting WACZs.

Reply:
Ah, it's not working because this isn't supported in js-wacz yet!